react/src/components/Weather/TodaysWeather/TodaysWeather.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component, useEffect, useState } from "react";
+import Weather from './TodaysWeather.module.css';
+import CurrentWeather from '../CurrentWeather/CurrentWeather';
+import { Dimmer, Loader } from 'semantic-ui-react';
+import {weatherApi} from '../Api';
+
+const TodaysWeather = () => {
+
+  const [lat, setLat] = useState([]);
+  const [long, setLong] = useState([]);
+  const [weatherData, setWeatherData] = useState([]);
+  const [Error, setError] = useState(null);
+
+  useEffect(() => {
+    navigator.geolocation.getCurrentPosition(function (position) {
+      setLat(position.coords.latitude);
+      setLong(position.coords.longitude);
+    });
+
+    getWeather(lat, long)
+      .then(weather => {
+        setWeatherData(weather);
+        setError(null);
+      })
+      .catch(err => {
+        setError(err.message);
+      });
+  }, [lat, long])
```

Comment:
[lat, long] zamiast tego chyba powinno być []?

Reply:
jak są puste klamry to nie wyświetla się w ogóle pogoda

Reply:
bo wtedy getWeather powinno być wywołane  po linijce 17 a przed 18 :)

Reply:
![Screen Shot 2022-07-25 at 11 57 22 AM](https://user-images.githubusercontent.com/99273662/180750828-019085ae-b342-495c-9636-15dce43928e1.png)

jeśli tak, to też nie działa 🥴

Reply:
a wyślij te zmiany proszę

Reply:
<img width="420" alt="image" src="https://user-images.githubusercontent.com/16018172/180761598-03c09193-1c85-49f5-afb8-399f61ac3522.png">